### PR TITLE
Completed DRY Employees.

### PR DESCRIPTION
**Pull request by ralphminor**

### CODE REVIEW

File: org_chart.js [hunk 1]
```diff
@@ -1,73 +1,58 @@
-let elon = {
-  name: "Elon Musk",
-  email: "elon.musk@musky.com",
-  position: "Boss",
+function Person(name, gender, email, skills) {
+  this.name = name;
+  this.gender = gender || "Not identified";
+  this.email = email || "none";
```
Comment (carlosnsr) [inline org_chart.js:4]: I like the checking for nulls and adding default values.
Best practice is to put these as default parameters in the parameter list.

File: org_chart.js [hunk 2]
```diff
@@ -1,73 +1,58 @@
...
-donald.manager = elon
-
-// send to training
-if (!alan.skills) {
-  alan.skills = []
+Person.prototype.hire = function (position, manager, project) {
+  this.position = (position || "TBD");
+  this.manager = (manager || "TBD");
+  this.project = (project || "no current projects");
 }
```
Comment (carlosnsr) [inline org_chart.js:12]: Try to stick to the [Single Responsibility Principle](https://en.wikipedia.org/wiki/Single_responsibility_principle) even in methods. This method is responsible for three things: setting the project, position,  and the manager. Break this into three methods, with descriptive names, that say what they do.

File: org_chart.js [hunk 3]
```diff
@@ -1,73 +1,58 @@
...
+  switch(this.gender) {
+    case "male":
+        salutation = "He";
+        break;
+    case "female":
+        salutation = "She";
+        break;
+    default:
+        salutation = this.name;
+  }
```
Comment (carlosnsr) [inline org_chart.js:29]: I like how you default to the name

File: org_chart.js [hunk 4]
```diff
@@ -1,73 +1,58 @@
...
-console.log(`She can be reached at ${grace.email}`)
-console.log(`${donald.name} is a ${donald.position} working on ${donald.project}.`)
-console.log(`He can be reached at ${donald.email}`)
-console.log(`${tim.name} is a ${tim.position} working on ${tim.project}.`)
-console.log(`He can be reached at ${tim.email}`)
+elon.describe();
+alan.describe();
+grace.describe();
+donald.describe();
+tim.describe();
```
Comment (carlosnsr) [inline org_chart.js:58]: Hiring, training, and describing a person could further be DRY'd

**Comment by carlosnsr** [comment]
Great job, Ralph.  Please respond to feedback and resubmit.

**Comment by carlosnsr** [comment]
Can you see my feedback now?

**Comment by ralphminor** [comment]
Got it.  Thanks.  I'll review and re-submit.

**Pull request closed without merging**